packages/app/metrics-utils/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@lokalise/metrics-utils",
+    "version": "v1.0.0",
+    "files": ["dist"],
+    "license": "Apache-2.0",
+    "type": "module",
+    "main": "./dist/index.cjs",
+    "module": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "homepage": "https://github.com/lokalise/shared-ts-libs",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/lokalise/shared-ts-libs.git"
+    },
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "require": "./dist/index.cjs",
+            "import": "./dist/index.js"
+        }
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "build": "rimraf dist && vite build",
+        "dev": "vite watch",
```

Comment:
this is not needed

Reply:
Adjusted to the format of other packages across this repo ✅